src/Services/GarbageCollectionService.php

```diff
@@ -2,14 +2,23 @@
 
 namespace SilverStripe\SessionManager\Services;
 
+use SilverStripe\Core\Config\Configurable;
 use SilverStripe\Core\Injector\Injectable;
+use SilverStripe\ORM\DB;
+use SilverStripe\ORM\FieldType\DBDatetime;
```

Comment:
```suggestion
```
I was going to ignore this but you need to resolve merge conflicts anyway so may as well remove this unnecessary statement at the same time.